typescript/src/schema/proto.ts

```diff
@@ -288,6 +289,23 @@ function toLRInterface(type: string, inputType: string) {
   return inputType.replace(/\.([^.]+)$/, '.I' + type);
 }
 
+function getHeaderParms(rule: plugin.google.api.IHttpRule): string {
```

Comment:
`Params`?

Reply:
Also, this function deserves a unit test in `test/unit/proto.ts`.

Reply:
yes, this is for getting parameters in the httprule, like `crypto_key.name` would be `[crypto_key, name]`, sure, adding unit test makes perfect sense.